src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -2,0 +1,12 @@
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+      {' by '}
+      <a className="CommentInfo__email" href={`mailto:${comment.email}`}>
```

Comment:
The `href` attribute for the email link is constructed correctly using a template literal. Ensure that `comment.email` is a valid email address to avoid issues with the mailto link.